processor/transformprocessor/internal/metrics/functions.go

```diff
@@ -18,6 +18,12 @@ var useConvertBetweenSumAndGaugeMetricContext = featuregate.GlobalRegistry().Mus
 	featuregate.WithRegisterDescription("When enabled will use metric context for conversion between sum and gauge"),
 )
 
+var useConvertExponentialHistogramToExplicitHistogram = featuregate.GlobalRegistry().MustRegister(
```

Comment:
Since a user must explicitly configure the function if they want the conversion, we do not need to block the feature via a feature gate.

Reply:
Thanks, Feature gate removed.